styles/landing.css

```diff
@@ -0,0 +1,109 @@
+@import url("./main.css");
+
+.products-nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.product {
+  font-weight: bold;
```

Comment:
This font-weight has been used multiple times so you can create a utility class for it named as txt-bold and use it wherever needed in the HTML file